src/VisualStudio/LiveShare/Impl/FindAllReferencesHandler.cs

```diff
@@ -20,6 +20,10 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.LiveShare
 {
+    /// <summary>
+    /// TODO - This should move to the ILanguageClient when we remove the UI thread dependency.
```

Comment:
Not sure this should be a `<summary>` tag -- if somebody mouses over this type, they probably don't want the TODO in quick info.